app/mailers/task_mailer.rb

```diff
@@ -0,0 +1,77 @@
+class TaskMailer < ApplicationMailer
+  def task_created(project, task)
+    @project_memberships = ProjectMembership.find_by(project_id: project.id)
+
+    @project_memberships.each do |member|
+      if member.role == "owner"
```

Comment:
Enumerize gave you a method `:owner` for best readable
```suggestion
      if member.owner?
```